OpenRA.Mods.Common/Widgets/ProductionTabsWidget.cs

```diff
@@ -105,6 +105,8 @@ public class ProductionTabsWidget : Widget
 		readonly Lazy<ProductionPaletteWidget> paletteWidget;
 		string queueGroup;
 
+		readonly Dictionary<ProductionQueue, bool> allProductionQueues = new();
```

Comment:
The name is not OK. It suggests a list/array/single-type collection and also dictionary naming is usually about the value.
Seeing how this is mostly for caching enabled state, I suggest `cachedProductionQueueEnabledStates`.